notebooks/tm/building_footprints/4_extract_features.ipynb

```diff
@@ -0,0 +1,851 @@
+{
```

Comment:
<p>All columns containing count of bldgs must be integers, kindly convert</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/lacuna-fund-climate-health/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>